src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4055,11 +4051,14 @@ components:
     StreamTransactionsResponse:
       allOf:
         - $ref: "#/components/schemas/LedgerStateMixin"
-        - $ref: "#/components/schemas/ResultSetCursorMixin"
         - type: object
           required:
             - items
           properties:
+            next_cursor:
```

Comment:
I take it we never return a `total_count`? So makes sense to not use the `ResultSetCursorMixin`, I agree

Reply:
yes it was never returned here and was optional.